### PR TITLE
On sign out - remove cookie and update news12 logo

### PR DESCRIPTION
**Pull request by syed-imam**

### CODE REVIEW

File: src/app/shared/components/header-side/header-side.component.ts [hunk 1]
```diff
@@ -133,4 +136,9 @@ export class HeaderSideComponent implements OnInit, OnDestroy {
       this.regionService.setFranklyRegion(this.selected);
   }
 
+  removeUserCookie()
+  {
+     localStorage.setItem("currentUser","");
+     this.router.navigate(['/sessions/signin']).then();
```
[the remark this answers is not in the document]
Comment (syed-imam) [reply src/app/shared/components/header-side/header-side.component.ts]: I can remove it, but IDE gives me a warning for ignoring the promise. And there isn't anything I want to do after the route is navigated to sign-in page, execution of code goes to sign in component.

[the remark this answers is not in the document]
Comment (kspector) [reply src/app/shared/components/header-side/header-side.component.ts]: @syed-imam in that case, maybe you can just `await` the promise? It seems like you're trying to do this action synchronously anyway

[the remark this answers is not in the document]
Comment (syed-imam) [reply src/app/shared/components/header-side/header-side.component.ts]: @kspector Perfect! awaiting on the promise returned from router.navigate

File: src/app/shared/components/sidebar-side/sidebar-side.component.ts [hunk 2]
```diff
@@ -56,4 +58,9 @@ export class SidebarSideComponent implements OnInit, OnDestroy, AfterViewInit {
           });
     }
   }
+
+  signOut() {
```
Comment (kspector) [inline src/app/shared/components/sidebar-side/sidebar-side.component.ts]: @syed-imam one more here

Comment (syed-imam) [reply src/app/shared/components/sidebar-side/sidebar-side.component.ts]: @kspector nice catch! :astonished:

**Comment by kspector** [comment]
@syed-imam i'll approve once merge conflicts are resolved

**Comment by syed-imam** [comment]
> @syed-imam i'll approve once merge conflicts are resolved

@kspector merged

**Pull request merged**